amundsen_application/static/js/ducks/search/sagas.ts

```diff
@@ -204,6 +204,10 @@ export function* urlDidUpdateWatcher(): SagaIterator {
 
 export function* loadPreviousSearchWorker(action: LoadPreviousSearchRequest): SagaIterator {
   const state = yield select(getSearchState);
+  if (state.search_term === "") {
```

Comment:
nit: add a test to verify `BrowserHistory.goBack()` is called? If for some reason this test is tricky to add I'm fine with `TODO` in the code because there's a backlog task to come back to covering some of the untested search saga logic.

Reply:
I didn't spend too much time but I couldn't figure out how to test that within the saga test.

Reply:
No problem, fine with being part of the overall unit test backlog task.